### PR TITLE
Feature/cmp 402   language switching loses context

### PR DESCRIPTION
**Pull request by makma**

### CODE REVIEW

File: Scripts/build.cake [hunk 1]
```diff
@@ -0,0 +1,159 @@
+#tool "nuget:?package=xunit.runner.console"
```
Comment (petrsvihlik) [inline Scripts/build.cake]: I believe the build scripts coming from DEVOPS-440 shouldn't be part of this PR. can you transfer your changes to a feature branch and submit a new PR? thx

Comment (makma) [inline Scripts/build.cake]: Of course you`re right

**Comment by petrsvihlik** [comment]
replaced with #26
